packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -398,8 +400,9 @@ class _HelperErrorState extends State<_HelperError> with SingleTickerProviderSta
     );
   }
 
+
   Widget _buildError() {
-    assert(widget.errorText != null);
+    assert(widget.errorText != null || widget.error != null);
```

Comment:
Looks like this assertion is causing failing tests.

Reply:
Ohhh so can you please tell me how can I put assertion for widget.error?

Reply:
Or can I remove the assertion?